meta-balena-common/recipes-support/resin-mounts/resin-mounts/resin-partition-mounter

```diff
@@ -1,121 +1,139 @@
 #!/bin/sh
 
+# Copyright 2019 Balena Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Tool for mounting a balenaOS partition
+#
+
 set -e
 
-LABEL_TIMEOUT=1900
+TIMEOUT_LOOPS=300 # ~ 100ms/loop * 300loops = 30000ms = 30s
 
+#
+# Help message
+#
 help () {
-	cat << EOF
-Mount a resin partition label
+    cat << EOF
+Tool for mounting a balenaOS partition
 $0 <ARGUMENTS>
 
 ARGUMENTS:
-	-h, --help
-		Print this message.
-	--mount <LABEL>
-		Action: Mount label
-	--umount <LABEL>
-		Action: Unmount label
+    -h, --help
+        Print this message
+    --mount <LABEL>
+        Action: Mount label
+    --umount <LABEL>
+        Action: Unmount label
+    --sysroot
+        Request the action on a sysroot label
 EOF
 }
 
 # Parse arguments
 while [ "$#" -ge 1 ]; do
-	i="$1"
-	case $i in
-		-h|--help)
-			help
-			exit 0
-			;;
-		--mount)
-			if [ -z "$2" ]; then
-				echo "ERROR: $1 argument needs a value."
-			fi
-			part_label="$2"
-			action="mount"
-			shift
-			;;
-		--umount)
-			if [ -z "$2" ]; then
-				echo "ERROR: $1 argument needs a value."
-			fi
-			part_label="$2"
-			action="umount"
-			shift
-			;;
-		--sysroot)
-			sysroot="yes"
-			;;
-		*)
-			echo "ERROR: Unrecognized option $1."
-			help
-			exit 1
-			;;
-	esac
-	shift
+    i="$1"
+    case $i in
+        -h|--help)
+            help
+            exit 0
+            ;;
+        --mount)
+            if [ -z "$2" ]; then
+                echo "ERROR: $1 argument needs a value."
+            fi
+            target="$2"
+            action="mount"
+            shift
+            ;;
+        --umount)
+            if [ -z "$2" ]; then
+                echo "ERROR: $1 argument needs a value."
+            fi
+            target="$2"
+            action="umount"
+            shift
+            ;;
+        --sysroot)
+            sysroot="yes"
+            ;;
+        *)
+            echo "ERROR: Unrecognized option $1."
+            help
+            exit 1
+            ;;
+    esac
+    shift
```

Comment:
the whitespace diffs make it hard to review..

Reply:
I fixed the whitespace to make it consistent with other files, could you use a tool that ignores whitespace for the review?

Reply:
I could have split the whitespace changes into a different commit. Will do next time.